README.md

```diff
@@ -318,7 +319,23 @@ sectors.name # Industrials
 sectors.performance # 0.00711
 sectors.last_updated # 1533672000437
 ```
-See [#sector-performance](https://iextrading.com/developer/docs/#sector-performance) for detaile documentation or [sectors.rb](lib/iex/resources/sectors.rb) for returned fields.
+See [#sector-performance](https://iextrading.com/developer/docs/#sector-performance) for detailed documentation or [sectors.rb](lib/iex/resources/sectors.rb) for returned fields.
+
+### Get Largest Trades
+
+Fetches largest trades.
+
+```ruby
+trades = IEX::Resources::LargestTrades.get('aapl')
+
+trades.price # 186.39
+trades.size # 10000
```

Comment:
This looks incorrect. The trades are a collection.

Reply:
Good catch! Looks like it returns a list of trades since the documentation shows an array json. Although, since now (Saturday Night) [nothing is being returned](https://api.iextrading.com/1.0/stock/aapl/largest-trades), looks like it will only return some data at daily trading hours. 

I'm gonna take a look at Monday after 9:45am (when NYSE starts trading again + 15 minutes delayed). Any fix now wouldn't allow a new VCR to record anything anyways.